source/src/index.html

```diff
@@ -41,8 +41,6 @@
         }
         #accessibility-content {
             padding: 0;
-            margin-left: 1.875vw;
```

Comment:
Make sure to update the html fragments directory with the same edits done to index.html.